zio-http-benchmarks/src/main/scala/zhttp.benchmarks/SchemeDecodeBenchmark.scala

```diff
@@ -0,0 +1,18 @@
+package zhttp.benchmarks
+import org.openjdk.jmh.annotations._
+import zhttp.http._
+
+import java.util.concurrent.TimeUnit
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+class SchemeDecodeBenchmark {
+  private val MAX = 1000000
+
+  @Benchmark
+  def benchmarkSchemeDecode(): Unit = {
+    (0 to MAX).foreach(_ => Scheme.unsafeDecode("HTTP"))
```

Comment:
```suggestion
    (0 to MAX).foreach(_ => Scheme.decode("HTTP"))
```

Let's compare the two implementations (old vs new).